train.py

```diff
@@ -35,11 +39,39 @@
 from utils.torch_utils import ModelEMA, select_device, intersect_dicts, torch_distributed_zero_first, is_parallel
 from utils.wandb_logging.wandb_utils import WandbLogger, check_wandb_resume
 
+# TODO: Rip out all Weights and Biases References
+# TODO: Rip out all GCP References
+# TODO: Rip out TensorBoard References?
+
 logger = logging.getLogger(__name__)
 
+# Get MLFlow basic auth credentials from AWS Secrets Store
+aws_secrets_client = boto3.client('secretsmanager', region_name="us-west-2")
+secrets = json.loads(aws_secrets_client.get_secret_value(SecretId="network/sandbox")["SecretString"])
```

Comment:
move all environment input to input (env vars or program input), so `network/sandbox` and `us-west-2` should be input.

Reply:
done